common/gc_top_menu/reports/pre-a11y-1.json

```
@@ -0,0 +1,462 @@
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:on-focus",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:on-input",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:consistent-navigation",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:consistent-identification",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:error-identification",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:labels-or-instructions",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:error-suggestion",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:error-prevention-legal-financial-data)",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:parsing",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:name-role-value",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:untested",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "earl:test": "WCAG21:status-messages",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    }
+  ]
+}
```

Every content file in this repository must end with an empty line. The red circle with a dash symbol illustrate that issue and it is also reported as an error by our lint spacing continuous integration test. You can also run that test locally but executing the CLI command `grunt test`

```suggestion
}

```